bar-trender-front-end/src/views/FilterResults.js

```diff
@@ -0,0 +1,53 @@
+import React, { useEffect, useState } from 'react';
+
+//css
+import  "./css/FilterResults.css"
+
+// core components
+import ExamplesNavbar from "../components/Navbars/ExamplesNavbar.js";
+import LandingPageHeader from "../components/Headers/LandingPageHeader.js";
+import List from "../components/List";
+import withListLoading from '../components/withListLoading';
+
+function ListPage() {
+  // Consuming REST GET
+  const ListLoading = withListLoading(List);
+  const [appState, setAppState] = useState({
+    loading: false,
+    repos: null,
+  });
+  useEffect(() => {
+    setAppState({ loading: true });
+    const apiUrl = `https://api.github.com/users/hacktivist123/repos`;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/BarTrender/BarTrenderFront/pullRequest?prid=7123646)
```suggestion
    const apiUrl = "https://api.github.com/users/hacktivist123/repos";
```